lib/smart_answer/calculators/coronavirus_find_support_calculator.rb

```diff
@@ -14,52 +14,72 @@ class CoronavirusFindSupportCalculator
                   :have_you_been_made_unemployed,
                   :mental_health_worries
 
-    def has_results?
-      return false if need_help_with.blank?
-
-      need_help_with != "none"
-    end
+    GROUPS = {
+      feeling_unsafe: [:feel_safe],
+      paying_bills: [:afford_rent_mortgage_bills],
+      getting_food: %i[afford_food get_food],
+      being_unemployed: %i[have_you_been_made_unemployed self_employed],
+      going_in_to_work: %i[worried_about_work are_you_off_work_ill],
+      somewhere_to_live: %i[have_somewhere_to_live have_you_been_evicted],
+      mental_health: [:mental_health_worries],
+    }.freeze
 
-    def needs_help_with?(given_help_item)
-      return false unless has_results?
+    SECTIONS = {
```

Comment:
I think `SECTIONS` is just adding complication to this class.

Defining all the calculator actions in one hash makes it more difficult to use and test each in isolation. It also prevents memory optimisation of each action. `SECTIONS` effectively replaces a standard ruby instance object with a complicated hash.